user-interface/src/lib/type-declarations/chapter-15.d.ts

```diff
@@ -47,6 +47,34 @@ interface CaseDetailType {
   debtorTypeLabel: string;
   petitionLabel: string;
 }
+export interface CaseAudit {
+  id?: string;
+  documentType: 'ASSIGNMENT';
+  caseId: string;
+  name: string;
+  role: string;
+  assignedOn: string;
+  unassignedOn?: string;
+}
+
+export interface CaseStaffAssignment {
```

Comment:
Why do CaseStaffAssignment and CaseAudit share the same definition? Same? Consolidate?